modules/tinymce/src/themes/silver/main/ts/ui/window/SilverInlineDialog.ts

```diff
@@ -100,7 +139,7 @@ const renderInlineDialog = <T>(dialogInit: DialogManager.DialogInit<T>, extra: S
         updateState,
         initialData: dialogInit
       }),
-      Focusing.config({ }),
+      Focusing.config({}),
```

Comment:
Why was the space here removed? You included in the other places previously (eg `BlockingDemo.ts`).

Reply:
Replaced. I don't know what my auto formatter does on a good day, tbh